.travis.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  - docker
+script:
+  - make test
+  - make package
+  - ls -alh nodejs-typescript-buildpack-*.tgz
+deploy:
+  provider: releases
+  api_key: $GITHUB_OAUTH_TOKEN
+  file:
+    - "nodejs-typescript-buildpack-${TRAVIS_TAG}.tgz"
+  skip_cleanup: true
```

Comment:
Why are we skipping cleanup?

Reply:
We want to skip cleanup because we want to release the build that was created by Travis and upload it to GitHub Releases.